src/app/right-hand-panel.js

```diff
@@ -63,22 +57,23 @@ export class RightHandPanel {
         $(this.element).find("a").click((e) => {
             e.preventDefault();
 
-            this.selected_tab = e.currentTarget.hash;
+            this.context.selected_tab = e.currentTarget.hash;
 
             // we don't allow clicking the regions if we don't show them
             // or if the regions info is not present
             let img_conf = this.context.getImageConfig(this.config_id);
-            if (this.selected_tab === '#rois' &&
+            if (this.context.selected_tab === '#rois' &&
                 (img_conf === null || img_conf.regions_info === null ||
                  img_conf.regions_info.data === null ||
                  img_conf.image_info.projection === 'split')) return;
 
-            if (!this.context.show_regions && this.selected_tab === '#rois') {
+            if (!this.context.show_regions && this.context.selected_tab === '#rois') {
                 this.context.show_regions = true;
                 this.context.publish(
                     REGIONS_SET_PROPERTY, {property: "visible", value: true});
             }
             $(e.currentTarget).tab('show');
+            console.log(this.context.selected_tab);
```

Comment:
Think you forgot about the console.log. btw: with 'npm run debug' you can also debug locally.